spec/hatchet/rails51_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+describe "Rails 5.1" do
+  it "works with webpacker + yarn (js friends)" do
+    Hatchet::Runner.new("rails51_webpacker").deploy do |app, heroku|
+      expect(app.output).to include("Installing yarn")
+      expect(app.output).to include("yarn install")
+      expect(app.run("rails -v")).to match("")
```

Comment:
We supposed to be expecting a rails version here?

Maybe we should check status of `yarn -v`?